data/pathfinder/paizo/player_companion/animal_archive/anar_abilities_class.lst

```diff
@@ -2,7 +2,7 @@
+Huntmaster		KEY:Cavalier Archetype ~ Huntmaster	CATEGORY:Archetype	TYPE:Archetype.CavalierArchetype.CavalierHeavyArmorProficiency.CavalierTactician.CavalierBanner.CavalierGreaterTactician.CavalierGreaterBanner.CavalierSupremeCharge.CavalierBonusFeats.CavalierMount																										PRECLASS:1,Cavalier=1	PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Cavalier Archetype ~ Huntmaster],[!PREABILITY:1,CATEGORY=Archetype,TYPE.CavalierBonusFeats,TYPE.CavalierHeavyArmorProficiency,TYPE.CavalierTactician,TYPE.CavalierBanner,TYPE.CavalierGreaterTactician,TYPE.CavalierGreaterBanner,TYPE.CavalierSupremeCharge]																															DESC:Huntmasters train the beasts favored by lordly castes into swift and deadly trackers.								ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Class Skills|PRECLASS:1,Cavalier=1	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Weapon and Armor Proficiency|PRECLASS:1,Cavalier=1	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Bestial Challenge|PRECLASS:1,Cavalier=1	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Hunting Pack|PRECLASS:1,Cavalier=1	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Tactician|PRECLASS:1,Cavalier=1	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Takedown|PRECLASS:1,Cavalier=3	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Animal Trainer|PRECLASS:1,Cavalier=4	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Swift Tracker|PRECLASS:1,Cavalier=5	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Bonus Feats|PRECLASS:1,Cavalier=6	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Pack Attack|PRECLASS:1,Cavalier=11	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Quarry|PRECLASS:1,Cavalier=14	ABILITY:Cavalier Class Feature|AUTOMATIC|Huntmaster ~ Improved Quarry|PRECLASS:1,Cavalier=20	COST:0	SOURCEPAGE:p.36	ASPECT:Archetype Base Class|Cavalier
```

Comment:
Ah, that would be my fault. Yeah, this works. I'll merge it in. 

We will definitely want the multi-companion support before we can call this issue resolved though. Bonus if you get the extra Companion supported by the Seekers of Secret feats, 'Boon Companion'. Look at the Beastmaster for how that was implemented.